app/views/spotlight/appearances/edit.html.erb

```diff
@@ -1,8 +1,9 @@
+<% jcrop_options = {croppable: true, selector: 'masthead_image', bg_color: 'black', bg_opacity: '.4', max_size: '[1800,131]', min_size: '[1200,120]'} %>
```

Comment:
Does it make sense to push these options into a helper and/or engine configuration?